scheduler/planner.go

```diff
@@ -236,6 +236,10 @@ func (unit *Unit) RankValue() int64 {
 		// waiting on the results), and because FIFO feels
 		// fair in this context.
 		unit.cachedValue += priority * unit.distro.GetPatchTimeInQueueFactor() * int64(math.Floor(timeInQueue.Minutes()/float64(length)))
+	} else if inCommitQueue {
+		// give commit queue patches a boost over everything else
+		priority += 200
```

Comment:
I think this makes https://github.com/evergreen-ci/evergreen/pull/3327/files#diff-eeb535db139364fdc1705631dc4dc155R219 redundant.